.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,6 +15,8 @@ Thanks for contacting us! Please read and follow these instructions carefully, t
 
 #### Description of expected behavior and the observed behavior
 
+- [ ] I'd like to create a pull request to address this
```

Comment:
```suggestion
- [ ] I may be interested in making a pull request to address this
```

Reply:
I was also thinking of adding another checkbox: `- [ ] I need step-by-step guidance`